### DISCUSSION TITLE
Sega Saturn Mouse testing

### DISCUSSION THREAD

**Discussion by mt09torrent** [darthcloud/BlueRetro#568] [Q&A]
Hello,
I can't make working saturn blueretro  with a bluetooth mouse. Somebody has a retex with virtua cop 2 for example ? I'm using [v1.82 hw1 saturn] FW and is working perfectly as gamepad and gamepadAlt with xbox one (legacy fw) but not with mouse (and default mouse preset) and 2 differents bluetooth hid mouse that seems to be connected fine to blueretro. If somebody can provide me advice :)

**Comment by darthcloud** [comment]
I usually test mouse with Myst on Saturn, so that known to work for sure.

Give it a try.

Sound like you got configured right but just in case check this:
https://github.com/darthcloud/BlueRetro/wiki/BlueRetro-BLE-Web-Config-User-Manual#81---mouse-config-example

**Reply by mt09torrent** [reply]
I tested Myst JP & US and check my config with the link you provide. But not working. My mouse was very old it's logitech M-RBB93.

**Reply by mt09torrent** [reply]
The mouse is working on my desktop PC and android phone. So, I have no idea how to check other way Saturn blueretro with mouse. Can someone tell me what mouse model is reported working ?

**Reply by mt09torrent** [reply]
Hello Again :) I just been made my mouse working. So I pre-selected option controller in virtua cop 2, and then set mouse for output 1 and 2 in blueretro advanced menu. And then I can see controller logo changed to mouse logo in VC2 option menu for both players. So I figured out that my mouse is registered as output 2 that's why my mouse was not working as intended previously.
Now I'm trying to get start bouton on mouse working because I only have A B and C. The wheel was not registred as start.

**Reply by mt09torrent** [reply]
Problem solved I found the solution with "Mapping Config" Web cli by changing dst label from default to mouse and then in dest column "M: Middle" by "4". So, that way with my mouse I can enjoy start bouton and I can continue Virtua Cop 2 instead of waiting count down to game over :) 
By defaut Blueretro config is using wheel label that I think is correct. So for older mouse don't hesitate to play with mapping config.
